timelock-impl/src/test/java/com/palantir/atlasdb/timelock/adjudicate/PointHealthReportAnalysisTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.adjudicate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import com.palantir.timelock.feedback.EndpointStatistics;
+
+public class PointHealthReportAnalysisTest {
+    private static final String CLIENT_1 = "Client1";
+
+    private static final double RATE =
+            Constants.LEADER_TIME_SERVICE_LEVEL_OBJECTIVES.minimumRequestRateForConsideration();
+    private static final long P_99 =
+            Constants.LEADER_TIME_SERVICE_LEVEL_OBJECTIVES.maximumPermittedP99().toNanos();
+    private static final double ERROR_PROPORTION =
+            Constants.LEADER_TIME_SERVICE_LEVEL_OBJECTIVES.maximumPermittedErrorProportion();
+
+    @Test
+    public void reportIsHealthyWhenEverythingIsRight() {
+        FeedbackHandler feedbackHandler = new FeedbackHandler();
+        EndpointStatistics statistics = EndpointStatistics
+                .builder()
+                .p99(P_99 - 1)
+                .oneMin(RATE + 1)
+                .errorRate(0)
+                .build();
+        PointHealthStatusReport healthStatusForMetric = feedbackHandler.getHealthStatusForMetric(
+                CLIENT_1,
+                statistics,
+                Constants.LEADER_TIME_SERVICE_LEVEL_OBJECTIVES);
+        assertThat(healthStatusForMetric.status()).isEqualTo(HealthStatus.HEALTHY);
+        assertThat(healthStatusForMetric.message().isPresent()).isFalse();
```

Comment:
nit: Pretty sure AssertJ has OptionalAsserts that operate directly on optionals, I'd prefer using those instead.